interfaces.go

```diff
@@ -55,21 +65,19 @@ type MapTable interface {
 // Multimap recipe
 //
 
-// MultimapTable lets you list rows based on a field equality, eg. 'list all sales where seller id = v'.
 type MultimapTable interface {
-	Set(v interface{}) Op
-	Update(v, id interface{}, m map[string]interface{}) Op
-	Delete(v, id interface{}) Op
-	DeleteAll(v interface{}) Op
-	List(v, startId interface{}, limit int, pointerToASlice interface{}) Op
-	Read(v, id, pointer interface{}) Op
-	MultiRead(v interface{}, ids []interface{}, pointerToASlice interface{}) Op
+	Set(value interface{}) Op
+	Update(value, id interface{}, updatedValues map[string]interface{}) Op
+	Delete(value, id interface{}) Op
```

Comment:
What exactly is the `id` here and in `Update()` above this?

Reply:
`id` can be the clustering key, I would suggest we change these so `value` is changed for `partitionKey` and `id` is changed for `clusteringKey`

Reply:
Could we also change `updatedValues` to `valuesToUpdate`?